src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesAccountModel.md

```diff
@@ -0,0 +1,114 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CognitiveServices.dll-Help.xml
+Module Name: Az.CognitiveServices
+online version: https://docs.microsoft.com/powershell/module/az.cognitiveservices/get-azcognitiveservicesaccountmodel
+schema: 2.0.0
+---
+
+# Get-AzCognitiveServicesAccountModel
+
+## SYNOPSIS
+Get Models available for a Cognitive Services account
+
+## SYNTAX
+
+### DefaultParameterSet (Default)
+```
+Get-AzCognitiveServicesAccountModel [[-ResourceGroupName] <String>] [-AccountName] <String>
```

Comment:
How about name Get-AzCognitiveServicesAccountModel to Get-AzCognitiveServicesAccount? And change AccountName to Name and keep AccountName as alias

Reply:
We have "Get-AzCognitiveServicesAccount"
"Get-AzCognitiveServicesAccountModel" is right now used for listing models available for this account.
Though it is uncertain right now, we do have potential to support individual Get in future, we want to reserve the "Name" parameter for this. Just like what we do in "Get-AzCognitiveServicesAccountDeployment" and "Get-AzCognitiveServicesAccountCommitmentPlan".

Reply:
Make sense.